<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FRLG Seed Seeker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/zapdos.ico</normaloff>:/icons/zapdos.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <widget class="QLabel" name="label32">
        <property name="text">
         <string>32-bit Seeds (one per line):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txt32bitSeeds"/>
      </item>
      <item>
       <widget class="QLabel" name="label16">
        <property name="text">
         <string>Optional 16-bit GBA Seeds (one per line):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="txt16bitFilter"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="frameLayout">
        <item>
         <widget class="QLabel" name="labelFrames">
          <property name="text">
           <string>Max Frames:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMaxFrames">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="singleStep">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCopyTable">
        <property name="text">
         <string>Copy Table to Clipboard</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableResults">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>32-bit Seed</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>16-bit Seed</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Frames</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnGenerate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
